src/components/Calculator.js

```diff
@@ -1,64 +1,55 @@
-    const { total, next, operation } = this.state;
-    this.keydown();
-    return (
-      <div className="main">
-        <div className="grid-container">
-          <div className="grid-itemResult">
-            <span>
-              {total}
+            {values.operation}
 
-              {operation}
-
-              {next}
-            </span>
-          </div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>AC</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>+/-</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>%</div>
-          <div role="button" className="grid-item operator" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>÷</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>7</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>8</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>9</div>
-          <div role="button" className="grid-item operator" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>x</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>4</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>5</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>6</div>
-          <div role="button" className="grid-item operator" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>-</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>1</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>2</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>3</div>
-          <div role="button" className="grid-item operator" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>+</div>
-          <div role="button" className="grid-zero" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>0</div>
-          <div role="button" className="grid-item" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>.</div>
-          <div role="button" className="grid-item operator" onKeyDown={this.keydown} tabIndex={0} onClick={this.compute}>=</div>
+            {values.next}
+          </span>
         </div>
+        <Button className="grid-item" text="AC" onClick={compute} />
+        <Button className="grid-item" text="+/-" onClick={compute} />
+        <Button className="grid-item" text="%" onClick={calculate} />
```

Comment:
- [ ] Great job, there is no `calculate` method in this component, maybe it's a typo kindly fix it. you got this 💪.